nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/FXMLController.java

```diff
@@ -622,13 +622,29 @@ void addFID(NMRData nmrData, boolean clearOps, boolean reload) {
         if (!reload) {
             getActiveChart().full();
             getActiveChart().autoScale();
-            chartProcessor.loadDefaultScriptIfPresent();
+            generateScriptAndParse(nmrData);
```

Comment:
The auto generate script will only happen if reload is false, but this is where the default script was checked for, so confirm if this is the correct spot?

Reply:
I think we should still load the script if it is present, and only generate if the script isn't there.

Think of this use case: 
1. You open nmrfx and a fid for the first time
2. This generates a default script (which is good, that's the change you made)
3. You change this script manually, click on process. This saves a process.py file in the FID directory
4. You close nmrfx and go back home
5. Next day, you open nmrfx. You want your changes from process.py to be loaded, not a new one being generated.